resources/local/config.toml

```diff
@@ -188,6 +182,21 @@ tarpit_duration = '10min'
 # legacy nodes running this software.
 tarpit_chance = 0.2
 
+# Weights for impact estimation of incoming messages, used in combination with
+# `max_incoming_message_rate_non_validators`.
+#
+# Any weight set to 0 means that the category of traffic is exempt from throttling.
+[network.estimator_weights]
+consensus = 0
+gossip = 0
+finality_signatures = 0
+deploy_requests = 1
+deploy_responses = 1
+block_requests = 1
+block_responses = 0
+trie_requests = 1
+trie_responses = 0
```

Comment:
There are many other `config.toml` files and I imagine they all need to be updated.

Reply:
I hope I got them all in https://github.com/casper-network/casper-node/pull/2645/commits/18dfdaa9985a00c19f034501d7efcb95fbc37e31